package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "start:dev": "webpack-dev-server --config webpack.config.dev.js --open --hot",
     "start": "npm run build && node dist/server.js",
+    "test": "jest",
```

Comment:
@ProfJigsaw thus looks good. Can we have a coverage flag after jest so that it generates test coverage reports? `jest --coverage`.